default_launch.json

```diff
@@ -26,6 +26,21 @@
         "options": {
           "manual_worker_connection_only": true
         }
+      },
+      {
+        "layer": "SimulatedPlayerCoordinator",
+        "points_of_interest": {
```

Comment:
Since SimulatedPlayerCoordinator'ss don't really have a position, why not make this a more readable loadbalancer? rectangle_grid or entity_id_sharding sound like a better candidate.